Formula/daemontools.rb

```diff
@@ -22,6 +22,30 @@ def install
     end
   end
 
+  plist_options :startup => true
+
+  def plist; <<~EOS
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+    <dict>
+      <key>Label</key>
+      <string>#{plist_name}</string>
+      <key>ProgramArguments</key>
+      <array>
+        <string>sh</string>
+        <string>-c</string>
+        <string>mkdir -p /service &amp;&amp; exec #{opt_bin}/svscanboot</string>
```

Comment:
We should definitely not create that directory from a service. It is part of the user's configuration, they create it. Very strong *no* to `mkdir -p /service`, we don't mess up with directories outside Homebrew.

Reply:
We should consider patching the need for this directory. Arguably it's not fully ported to macOS if it requires it.

Reply:
We don't want to patch djb software. It really does not follow the Unix norms, so if we start it will be endless

Reply:
🆒 (although defining "djb" would be useful for my own learnings)

Reply:
Okay. I removed `mkdir -p /service` from plist and add caveats at 69de5fd1de73.